content/docs/15.how-to-guides/synchonous-executions-api.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Synchronous Executions API
+icon: /docs/icons/tutorial.svg
+---
+
+There are use-cases wher you want to trigger the flow, and get the flow's output in the API's response. In other words, you want the Executions API to behave synchronously.
+
+## Executions API
+
+Firstly, let us take a quick look at the Executions API. Executions API is an API used to invoke a flow execution. Say you have the following flow:
+
+```yaml
+id: myflow
+namespace: company.team
+tasks:
+  - id: mytask
+    type: io.kestra.plugin.core.debug.Return
+    format: hello from kestra
+
+outputs:
+  - id: mydata
+    type: STRING
+    value: "{{ outputs.mytask.value }}"
+    description: return some data
+```
+
+You invoke this flow using the Executions API as follows:
+
+```
+% curl -X POST  http://localhost:8080/api/v1/executions/company.team/myflow
+```
+
+By default, the Executions API is asynchronous. It will invoke the execution of the flow, and return back with only the time at which the execution was invoked. The response you will get from the above API call will look like:
+
+```json
+{
+  "id": "1KWLxLeaXEXNDaXWP7YSKA",
+  "namespace": "company.team",
+  "flowId": "myflow",
+  "flowRevision": 1,
+  "state": {
+    "current": "CREATED",
+    "histories": [
+      {
+        "state": "CREATED",
+        "date": "2024-07-12T05:07:28.447110427Z"
+      }
+    ],
+    "duration": "PT0.002939292S",
+    "startDate": "2024-07-12T05:07:28.447110427Z"
+  },
+  "originalId": "1KWLxLeaXEXNDaXWP7YSKA",
+  "deleted": false,
+  "metadata": {
+    "attemptNumber": 1,
+    "originalCreatedDate": "2024-07-12T05:07:28.447113302Z"
+  }
+}
+```
+
+## Synchronous Executions API
+
+In case you would like the API to wait till the execution is complete, and return back with the outputs from the flow in the response, you can call the Executions API with `wait=true` query parameter. This would make the API call synchronous, and you will receive all the outputs in the response that are explicitly exposed in the flow. You can invoke the Executions API in synchronous fashion as follows:
```

Comment:
```suggestion
In order to wait for an execution to finish and return the outputs from the flow in the response, you can call the Executions API with the `wait=true` query parameter. This would make the API call synchronous, and you will receive all outputs in the response that are explicitly exposed in the flow. You can invoke the Executions API in a synchronous fashion as follows:
```